website/pages/en/developing/graph-ts/api.mdx

```diff
@@ -536,6 +536,31 @@ For more information:
 - Encoding/decoding [Rust library/CLI](https://github.com/rust-ethereum/ethabi)
 - More [complex example](https://github.com/graphprotocol/graph-node/blob/6a7806cc465949ebb9e5b8269eeb763857797efc/tests/integration-tests/host-exports/src/mapping.ts#L72).
 
+#### Balance of an Address
+
+The balance of an address can be retrieved using the `ethereum` module. This feature is available from `apiVersion: 0.0.9` defined `subgraph.yaml`. The `getBalance()` retrieves the balance of the specified address from the block currently being processed within the subgraph.
```

Comment:
I think one nuance to note is that this gives the balance as of the end of the block, rather than at the moment the event is executed

Reply:
Also small update on wording:

 This feature is available from `apiVersion: 0.0.9`, which is defined in the `subgraph.yaml` manifest file.